webapp/src/app/models/Material.ts

```
@@ -9,9 +9,10 @@ export interface Material {
   description?:string
   courseId?:string;
   channelId?:string;
+  showDialog?:Boolean;
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)